lib/storage.js

```diff
@@ -83,7 +66,7 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
  *
  * @return {Promise<Object>} resolves to the database record.
  */
-authorizations.insert = async ({controller, capability}) => {
+authorizations.insert = async ({controller, capability} = {}) => {
```

Comment:
Do the authorizations and zcap APIs and collections need to be removed entirely?

Reply:
Not right now, no. Those are still used. We may remove `authorizations` in the future to force there to be just one way to manage which zcaps can be invoked but `zcaps` is used as the place to store zcaps that have been delegated to *you*. That will only go away if we likewise replace it with some other way of doing that.

Reply:
OK, I think we should rename the actual mongo collection `zcap-authorization` per https://github.com/digitalbazaar/bedrock-zcap-storage/issues/1

And I guess the `zcap` collection can just stay as is.

I don't know if this PR was a breaking change, but if we do this, it certainly will be.

@dlongley sound good?